src/Platform/ACSFPlatform.php

```diff
@@ -44,6 +47,11 @@ class ACSFPlatform extends PlatformBase implements PlatformSitesInterface, Platf
 
   public const SITEFACTORY_TOKEN = 'acquia.acsf.token';
 
+  public const GROUP_CONFIG_LOCATION = [
```

Comment:
I have just realised it is not platform specific config but true for all the platforms. Let's move it into PlatformBase and do not define it in multiple places.